frontend/src/metabase/visualizations/components/legend/ChartWithLegend.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import _ from "underscore";
+import ExplicitSize from "metabase/components/ExplicitSize";
+import {
+  ChartContent,
+  ChartWithLegendRoot,
+  LegendContent,
+} from "./ChartWithLegend.styled";
+import Legend from "./Legend";
+
+const MIN_WIDTH_PER_SERIES = 100;
+const MIN_UNITS_PER_LEGEND = 6;
+
+type Props = {
```

Comment:
Even though we have flow types somewhere, we don't use them anymore. We kept existing ones to have at least some information about some complex parts like `metabase-lib`. Now, we do use PropTypes which are not so great due to runtime type checking, however, they increase readability a bit and they can be converted to typescript types using codemods